sdk/spring/spring-cloud-azure-starter-monitor/README.md

```diff
@@ -93,7 +93,7 @@ You can then configure the connection string in two different ways:
 
 #### Cloud role name
 
-The [Application Map](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-map?tabs=net#set-or-override-cloud-role-name) uses the cloud role name to identify components on the map.
+The [Application Map](https://learn.microsoft.com/azure/azure-monitor/app/app-map?tabs=net#set-or-override-cloud-role-name) uses the cloud role name to identify components on the map.
```

Comment:
The CI requires to remove the local.

Reply:
which CI, can you please share the link?

Reply:
See https://dev.azure.com/azure-sdk/public/_build/results?buildId=3328581&view=logs&jobId=36df4387-3551-57fa-098c-b43a23930261&j=36df4387-3551-57fa-098c-b43a23930261&t=2c6e0737-9c89-5837-e57c-a71fe04a614a 

![image](https://github.com/Azure/azure-sdk-for-java/assets/14811066/7cd9094a-5afc-44b9-b77c-4856c31eaa35)

Reply:
> this link will get redirected to the old link though. what is the point of updating it?

for non en-us users it will be redirected to a different link